src/server.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Consider using ES modules with import statements instead of require for better interoperability and future-proofing your code 🚀